test/unit/selector.js

```diff
@@ -261,7 +279,7 @@ QUnit.test( "disconnected nodes", function( assert ) {
 	assert.equal( $div.is( "div" ), true, "Make sure .is('nodeName') works on disconnected nodes." );
 } );
 
-QUnit.test( "disconnected nodes - jQuery only", function( assert ) {
+QUnit[ jQuery.find.compile ? "test" : "skip" ]( "disconnected nodes - jQuery only", function( assert ) {
```

Comment:
I think all the `jQuery only` suffixes can be dropped, since they were only here as a poor man's `QUnit.skip`.